docs/guides/docker.md

```diff
@@ -0,0 +1,256 @@
+# section from the Wireguard container and add it to the Transmission and SABnzbd containers. The unused "default Wireguard listening port"
+# line can also be safely removed.
+#
+# 2. Docker Compose's `volume parameter`
+#
+# The `volume` parameter lets you mount directories from your host system to the Docker container so that information can persist upon bringing a container
+# down or up.
+#
+# Following the recommend directory structure from before, we've already filled out the volume mount settings, so that every service will have it's
+# own directory for it's compose files, all in the form of `/home/user/projects/automation/<service name>`.
+#
+# This can of course be changed if you're experienced with Docker and would prefer to use custom mount locations.
+#
+# 3. Values for PUID and PGID
+#
+# These values refer to the host OS user's UID and GID, respectively. You can find this out by entering `id $user` in your terminal.
+#
+# 4. Timezones
+#
+# All instances of `TZ` in the `environment` section should be changed to `TZ=Your/Timezone`. To find your timezone and the correct way to format it
+# for Docker, you can check this Wikipedia article (https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List) and get the information
+# from the "TZ identifier" column
+
+---
+version: "3.8"
+services:
+  wireguard:
+    container_name: wireguard
+    image: linuxserver/wireguard
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/London
+    cap_add:
+      - NET_ADMIN
+      - SYS_MODULE
+    sysctls:
+      - net.ipv4.conf.all.src_valid_mark=1
+      - net.ipv6.conf.all.disable_ipv6=0
+    ports: # Port openings are handled through the VPN container when a VPN sits in front of the container.
+      - '51820:51820/udp' # Default WireGuard listening port
+      - '8080:8080' # SABnzbd UI
+      - '9091:9091' # Transmission ui
+	  - '7474:7474' # Autobrr
+    dns:
+      - 1.1.1.1
+    volumes:
+      - './wireguard:/config'
+      - '/lib/modules:/lib/modules' # Don't touch this unless you know what you're doing!
+    networks:
+      - guide
+    restart: unless-stopped
+  sabnzbd:
+    container_name: sabnzbd
+    image: lscr.io/linuxserver/sabnzbd:latest
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/London
+    depends_on:
+      - wireguard
+    volumes:
+      - './sabnzbd:/config'
+      - '/home/user/data/usenet:/downloads'
+    network_mode: "service:wireguard"
+    restart: unless-stopped
+  transmission:
+    image: lscr.io/linuxserver/transmission:latest
+    container_name: transmission
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Europe/London
+      - PEERPORT=1000 # Replace this with the port that you forwarded during the `Generating a Wireguard configuration file` step.
+      - TRANSMISSION_RPC_PORT=9091
+    depends_on:
+      - wireguard
+    volumes:
+      - './transmission:/config'
+      - '/home/user/data/torrents:/downloads'
```

Comment:
this is bad, breaks hardlinking. The same parent folder (e.g `/home/user/data/:/data` must be mounted across your automation stack. You'll do the differentiating (`/usenet`, `/torrents`, etc) in the Web UI

Reply:
fixed and made the volume mounts consistent across all containers.